lib/components/Select.tsx

```diff
@@ -10,10 +10,19 @@ interface Option {
 interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
   name: string;
   label?: string;
+  placeholder?: string;
   options: Option[];
 }
 
-export default function Select({ name, label, options, ...rest }: Props) {
+export default function Select({
+  name,
+  label,
+  placeholder,
+  options,
+  ...rest
+}: Props) {
+  const defaultPlaceholderValue = "";
+  const defaultPlaceholderText = "";
```

Comment:
I'd vote to use `defaultProps` instead of these two variables.

Reply:
Hi @arthurdenner, thanks for your feedback.

I created these two variables to improve the readability only. Both, the default placeholder `value` and `text` are an internal behavior of the `Select` component and this behavior should not be controlled by `props`.

Reply:
Thanks @cleitonper, but I didn't understand what you meant.

You're creating two variables that will only be used if you don't receive the `placeholder` and the `defaultValue` props. They're fallback values, right?

You can have the same behaviour without any extra variables just putting these two props with empty strings as their `defaultProps`.

Reply:
@arthurdenner I make some changes based on your suggestions. Tell me what you think about that.

Reply:
It's better, but you can improve it by using the [`defaultProps`](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) feature of React itself instead of default values when deconstructing the props.

This would improve the readibility (in my opinion) and would also solve your problem related to test coverage related below.

Reply:
It's done. Thanks (;

Reply:
You're welcome! I hope to have helped in any way. :smile: